src/ripple/json/impl/json_value.cpp

```diff
@@ -1068,6 +1068,24 @@ Value::get ( std::string const& key,
 }
 
 Value
+Value::removeElement ( std::uint32_t index )
+{
+    JSON_ASSERT (type_ == arrayValue || index < value_.map_->size ());
```

Comment:
I don't know about this `removeMember` we need to study the JSON code and make sure it can't break invariants.